src/Server/Services/Analytics/AnalyticsService.lua

```diff
@@ -0,0 +1,575 @@
+--!strict
+
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+local MarketplaceService = game:GetService("MarketplaceService")
+
+local Packages = ReplicatedStorage.Packages
+local Knit = require(Packages.Knit)
+local GameAnalytics = require(Packages.GameAnalytics)
+
+local CURRENCIES = { "Coins" } -- List of all in-game currencies
```

Comment:
Constants and other variables should have their types defined (Eg. CURRENCIES: {string} = {"Coins"}